workflows/PostAssembly.nf

```diff
@@ -91,6 +92,12 @@ workflow POST_ASSEMBLY {
         log.info "No subtyping of assemblies performed"
     }
 
+
+    if(!params.skip_allele_calling && (!params.skip_species_classification || params.allele_scheme)){
```

Comment:
The logic here might be a bit easier if it was broken up into two `if` statements. We can then also more easily print specific logging messages if locidex is not run. That is:

```nextflow
if (!params.skip_allele_calling) {
    if (params.allele_scheme || !params.skip_species_classification) {
        // run locidex here
    } else {
        log.info "Skipping locidex since there is no '--allele_scheme' set and '--skip_species_classification' is enabled"
    }
}
```

Reply:
fixed in: be8cd5bdd64c18915c8a90accba7d58d6bedc7d3